graph/src/data/subgraph/mod.rs

```diff
@@ -604,6 +611,43 @@ pub struct Mapping {
     pub link: Link,
 }
 
+impl Mapping {
+    pub fn calls_host_fn(&self, host_fn: &str) -> bool {
```

Comment:
We probably want to call this a few times for each mapping, so it may make sense to have a method that returns the parsed payloads that we can then run multiple checks on.

Something like
```rust
struct ParsedMapping (Vec<Payload>);

impl ParsedMapping {
    fn calls_host_fn(&self, host_fn: &str) -> bool {
        ...
    }
}

impl Mapping {
    fn parse(&self) -> Result<ParsedMapping, Error> {
        ...
    }
}
```